### ISSUE TITLE
the tag <the-mask /> is mandatory?

### ISSUE THREAD

**Issue by Tecayehuatl** [vuejs-tips/vue-the-mask#85]
I want to know if I can use this with a simple <input v-directive/> or similar, I can't see anything related in documentation, or any topic that talk about it, this is because I have input templates, so I don't need this <the-mask /> tag, thank you!

**Comment by marcosrocha85** [comment]
According to the [docs](https://vuejs-tips.github.io/vue-the-mask/), you can use like this:
```<input type="tel" v-mask="'##/##/####'" />```

**Comment by Tecayehuatl** [comment]
Yeah, but I didn't want to use static masks, my solution was to use [inputmask](https://github.com/RobinHerbots/Inputmask) lib and create my own vue directive for non static masks, (not jQuery needed)
